app/assets/javascripts/editor.js

```diff
@@ -23,131 +29,119 @@ $E = {
         return code;
       }
     });
-  },
-  setState: function(textarea = 'text-input', preview = 'comment-preview-main', title = 'title') {
-    $E.title = $('#' + title + 'title'); // not sure why this exists? seems like $E.title is always #title
-    $E.textarea = $('#' + textarea);
+  }
+  setState(commentFormID) {
```

Comment:
Removing the default parameter (`commentFormID = "main"`) here. I think it makes sense so that errors will be easier to catch.